tests/test_jet_swapper.py

```
@@ -0,0 +1,70 @@
+    def test_data_shape(self):
+        result = self.swapper.data(self.df)
+        self.assertEqual(result.shape, (10, 18))
+
+    def test_data_type(self):
+        result = self.swapper.data(self.df)
+        self.assertIsInstance(result, torch.Tensor)
+        self.assertEqual(result.dtype, torch.float)
+
+    def test_swap_logic(self):
+        self.swapper.data(self.df)
+        mask = self.swapper.mask
```

Ruff

Local variable `mask` is assigned to but never used

Remove assignment to unused variable `mask`

(F841)